src/audio_decoder_midi.h

```diff
@@ -210,6 +220,9 @@ class AudioDecoderMidi final : public AudioDecoderBase, public midisequencer::ou
 	// Contains one entry per tempo change (latest on top)
 	// When looping all entries after the loop point are dropped
 	std::vector<MidiTempoData> tempo;
+
+	void SetLogVolume();
+	int midi_requested_channel_pans[16] {64, 64, 64, 64, 64, 64, 64, 64, 64, 64, 64, 64, 64, 64, 64, 64};
```

Comment:
To be consistent with volume: call the variable ``std::array<uint8_t, 16> channel_pans``